.travis.yml

```diff
@@ -43,10 +48,9 @@ before_install:
   # Set up the Groovy environment
   - export GROOVY_HOME=/usr/share/groovy/
 
-cache:
-  directories:
-    - $HOME/android-ndk-linux
-    - $HOME/android-sdk-linux
```

Comment:
Doesn't seem to make it take too long:

https://travis-ci.org/facebook/buck/builds/229837080?utm_source=github_status&utm_medium=notification